### PR TITLE
Gives artificers feedback when healing other constructs plus a few sprite fixes

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
rscadd: Artificers now have feedback when healing other constructs, showing how much health the target has.
tweak: Constructs can move in space.
imageadd: Fixes cult flooring not lining up with normal floors.
imageadd: Fixes a juggernaut back spine not glowing where it should.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/constructs.dm [hunk 1]
```diff
@@ -54,9 +56,19 @@
...
+				M.visible_message("<span class='danger'>[M] repairs some of \the <EM>[src]'s</EM> dents.</span>", \
+						   "<span class='cult'>You repair some of <EM>[src]'s</EM> dents, leaving <EM>[src]</EM> at [health]/[maxHealth] health.</span>")
+			else
+				M.visible_message("<span class='danger'>[M] repairs some of its own dents.</span>", \
+						   "<span class='cult'>You repair some of your own dents, leaving you at [M.health]/[M.maxHealth] health.</span>")
+		else
+			if(src != M)
+				M << "<span class='cult'>You cannot repair <EM>[src]'s</EM> dents, as it has none!</span>"
+			else
+				M << "<span class='cult'>You cannot repair your own dents, as you have none!</span>"
```
Comment (Anonus) [inline code/modules/mob/living/simple_animal/constructs.dm]: This entire thing is pretty bad is there a way I can make it simpler somehow

**Comment by KorPhaeron** [comment]
>constructs can move in space.

They already could though git gud

File: code/modules/mob/living/simple_animal/constructs.dm [hunk 2]
```diff
@@ -14,6 +14,8 @@
 	stop_automated_movement = 1
 	status_flags = CANPUSH
 	attack_sound = 'sound/weapons/punch1.ogg'
+	environment_smash = 1
+	see_in_dark = 7
```
Comment (phil235) [inline code/modules/mob/living/simple_animal/constructs.dm]: undocumented features?

File: code/modules/mob/living/simple_animal/constructs.dm [hunk 3]
```diff
@@ -54,9 +56,19 @@
 /mob/living/simple_animal/construct/attack_animal(mob/living/simple_animal/M)
 	if(istype(M, /mob/living/simple_animal/construct/builder))
 		adjustBruteLoss(-5)
```
Comment (phil235) [inline code/modules/mob/living/simple_animal/constructs.dm]: you could move `adjustBruteLoss(-5)` after `if(health < maxHealth)`

**Pull request merged**